platform/components/CustomNavbar/styles.module.css

```diff
@@ -0,0 +1,36 @@
+.container {
```

Comment:
Please only use tailwind css which can be given in className attribute of a html tag.